Sources/SKSwiftPMWorkspace/SwiftPMBuildSystem.swift

```diff
@@ -544,7 +544,7 @@ extension SwiftPMBuildSystem: SKCore.BuildSystem {
       return targets.map(ConfiguredTarget.init)
     }
 
-    if path.basename == "Package.swift" {
+    if path.basename == "Package.swift" && AbsolutePath(projectRoot) == path.parentDirectory {
```

Comment:
Would it be possible to add a test case for the issue?

Reply:
Sure thing, I've added a test case.